bika/lims/browser/analysisrequest/templates/analysisrequests.pt

```
@@ -1,87 +1,67 @@
-                   tal:condition="python:len(view.context_actions) > 0"
-                   tal:attributes="
-                      value python: view.getDefaultAddCount()"
-            />
-            <a tal:attributes="
-				href python:view.context_actions[add_item]['url'];
-				class python:'context_action_link %s %s' % (view.context_actions[add_item].get('class',''), 'ar_count')"
-               style="padding:5px 1px 3px 1px;"
-               tal:condition="python:len(view.context_actions) > 0">ARs
-            </a>
-        </tal:add_actions>
-    </h1>
-    <script type="text/javascript">
-	(function( $ ) {
-	$(document).ready(function(){
-		$('input.ar_count').change(function(){
-			url = $("a.ar_count.context_action_link")
-                .attr('href').split("?")[0];
-			$("a.ar_count.context_action_link")
-                .attr('href', url + "?ar_count="+$(this).val());
-		});
-		$('input.ar_count').click(function(e){e.preventDefault();})
-		$('input.ar_count').change();
-	});
-	}(jQuery));
-    </script>
-</metal:content-title>
+             tal:attributes="src view/icon"
+             style="height:32px;width:auto;"/>
+        <span tal:content="python:view.context.translate(view.title)"/>
+        <span style="padding-left:0.5em;display:inline-block;">
+          <form method="GET" class="form-inline" action="ar_add">
+            <input type="number"
+                   name="ar_count"
+                   min="1"
+                   style="width:4em;vertical-align:middle;"
+                   class="form-control input-sm"
+                   tal:attributes="value view/getDefaultAddCount|1"/>
+            <button type="submit"
+                    class="btn btn-link">
+              <img src="#"
+                   style="height:16px;width:auto;"
+                   tal:attributes="src string:${context/absolute_url}/++resource++bika.lims.images/add.png"/>
+              <span i18n:translate="">Add</span>
+            </button>
+          </form>
```

I was thinking that we could use Plone's "editContentActionMenus" to render these "Add" buttons instead. Don't think is necessary to display the number of ARs to be created here since that number can be set in AR Add form already. Also, worth to say that you will not be able to create more than 7 or 8 ARs in AR add form because the POST/GET header happens to be too large (at least, for nginx with default configuration), and the submission will fail silently.